ops/ansible/playbooks-ccs/ansible.cfg

```diff
@@ -42,4 +42,4 @@ pipelining = True
 #ssh_args = -o ControlMaster=no
 
 [inventory]
-enable_plugins = ini
+#enable_plugins = ini
```

Comment:
Had to do this in order to get the `--inventory=foo.ec2.internal` play parameter working, as it requires the enabled-by-default-but-disabled-here `hosts_list` plugin to be enabled.

Anyone know why this config was locked down to just `ini`? If there's not a good reason, let's just deleted this line altogether as part of this PR.

Reply:
No idea why it was locked to ini. Agreed we should remove.